examples/web-instrumentation/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "end-to-end-instrumentation-browser",
+  "description": "Web instrumentation demo",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "scripts": {
+    "lint": "eslint -c .eslintrc.js 'src/**/*.js' --fix",
+    "start": "node ./src/app.js"
+  },
+  "dependencies": {
+    "@opencensus/exporter-stackdriver": "0.0.19",
+    "@opencensus/nodejs": "0.0.19",
+    "@opencensus/propagation-tracecontext": "0.0.19",
```

Comment:
Just curious to know, why we are using both OpenCensus and OpenTelemetry. Stackdriver exporter is now available under OpenTelemetry along with trace context propagator.

Reply:
I am giving this a try. See latest code push. It is still not working yet.

Reply:
Made some more changes. This is working now.